### DISCUSSION TITLE
App for Android SATPI

### DISCUSSION THREAD

**Discussion by tomatos666** [Barracuda09/SATPI#137] [Q&A]
Hello wich APP from the Playstore are recommended for use with he SATPI.
I found no App that work well. Only PRGDVB find some channels but it chrash every time when it finisch with the channel search and the channel list are allways empty...

**Comment by Barracuda09** [comment]
Hello,

Are you using DVB-C or S(2)

I use sometimes Aurora TV.

**Reply by tomatos666** [reply]
I use DVB-T

**Reply by Barracuda09** [reply]
I am not sure 'Aurora TV' will do DVB-T
